Apps/Admin/Tests/Functional/cypress/integration/e2e/agentaccess.cy.js

```diff
@@ -0,0 +1,139 @@
+const username = Cypress.env("idir_username");
+const password = Cypress.env("idir_password");
+
+describe("Provision", () => {
+    beforeEach(() => {
+        cy.login(
+            Cypress.env("keycloak_username"),
+            Cypress.env("keycloak_password"),
+            "/provision"
+        );
+    });
+
+    it.only("Create, search, update, delete user.", () => {
```

Comment:
This doesn't adhere to the typical set of tests that are validating discrete functionality.

Reply:
Same as communications.cy.js